diffrax/_adjoint.py

```diff
@@ -852,3 +852,38 @@ def loop(
         )
         final_state = _only_transpose_ys(final_state)
         return final_state, aux_stats
+
+
+class ForwardMode(AbstractAdjoint):
+    """Differentiate through a differential equation solve during the forward pass.
```

Comment:
More precisely, this _enables support for forward-mode autodifferentiation_. (Not that it does the autodiff for you itself.)

Reply:
Right! :D